### PR TITLE
Typo in git-commit at git-branch description

### PR DESCRIPTION
**Pull request by huecester**
## Overview
Changed `[banch-name]` to `[branch-name]` under Related Terms

### CODE REVIEW

**Comment by brianamarie** [comment]
Thank you for this fix, @huecester!

**Review by kkash18 — APPROVED** [review]
OpenShift Container

File: git-guides/git-clone.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ If you choose to clone with SSH, you would use a specific SSH path for the repos
 
 ## Related Terms
 
-- `git branch`: This shows the existing branches in your local repository. You can also use `git branch [banch-name]` to create a branch from your current location, or `git branch --all` to see all branches, both the local ones on your machine, and the remote tracking branches stored from the last `git pull` or `git fetch` from the remote.
```
Comment (kkash18) [inline git-guides/git-clone.md:64]: ``

File: git-guides/git-clone.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ If you choose to clone with SSH, you would use a specific SSH path for the repos
...
+- `git branch`: This shows the existing branches in your local repository. You can also use `git branch [branch-name]` to create a branch from your current location, or `git branch --all` to see all branches, both the local ones on your machine, and the remote tracking branches stored from the last `git pull` or `git fetch` from the remote.
```
Comment (kkash18) [inline git-guides/git-clone.md:64]: []()

File: git-guides/git-clone.md (hunk 1 above)
Comment (kkash18) [inline git-guides/git-clone.md:64]: git-guides/git-clone.md@github-cla

**Review by kkash18 — CHANGES_REQUESTED** [review]

**Review by kkash18 — APPROVED** [review]

**Pull request merged**